migrations/20190919-initdb-builds.js

```diff
@@ -52,10 +52,10 @@ module.exports = {
                     type: Sequelize.TEXT
                 },
                 parameters: {
-                    type: Sequelize.TEXT
+                    type: Sequelize.TEXT('medium')
```

Comment:
if the dialect doesn't support, same as https://github.com/screwdriver-cd/datastore-sequelize/pull/58/files#r328474936